crates/common/tedge_config/src/tedge_config_dto.rs

```diff
@@ -72,6 +72,9 @@ pub(crate) struct CumulocityConfigDto {
 
     /// Boolean whether Azure mapper adds timestamp or not.
     pub(crate) mapper_timestamp: Option<bool>,
+
+    /// Set of c8y templates used for subscriptions.
+    pub(crate) smartrest_templates: Option<TemplatesSet>,
```

Comment:
Now that `TemplatesSet` is nothing more than a `Vec<String>`, is it even relevant? Can't this `smartrest_templates` just be:
```suggestion
    pub(crate) smartrest_templates: Option<Vec<String>>,
```